collect_app/src/main/java/org/odk/collect/android/widgets/AudioWidget.java

```diff
@@ -174,10 +203,9 @@ public void onClick(View v) {
                             getContext().getString(R.string.activity_not_found,
```

Comment:
Should use `showShortToast` from ToastUtils class instead of normal Toast builder